packages/teams-js/src/public/meeting.ts

```diff
@@ -334,6 +334,47 @@ export namespace meeting {
      * @returns A promise with the updated microphone state
      */
     micMuteStateChangedCallback: (micState: MicState) => Promise<MicState>;
+    /**
+     * Callback for the host to tell the app to change its speaker selection
+     */
+    audioDeviceSelectionChangedCallback?: (selectedDevices: AudioDeviceSelection) => void;
+  }
+
+  /**
+   * Interface for AudioDeviceSelection from host selection
+   *
+   * @hidden
+   * Hide from docs.
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   *
+   * @beta
+   */
+  export interface AudioDeviceSelection {
+    speaker?: AudioDeviceInfo;
+    microphone?: AudioDeviceInfo;
+    error?: SdkError;
+  }
+
+  /**
+   * Interface for AudioDeviceInfo, includes a device label with the same format as `MediaDeviceInfo.label`
```

Comment:
Can we add @link for `MediaDeviceInfo.label` same for other mentioned in the comment.

Reply:
Updated